220080/문제1.py

```diff
@@ -0,0 +1,23 @@
+# 최수빈 구하기
+
+import sys
+sys.stdin = open('input.txt','r')
+
+T = int(input())
+d = dict()
+a = []
+for t in range(1,T+1):
+    d = dict()
+    num = int(input())
+    num1 = list(map(int,input().split()))
+    for i in num1:
+        if i not in d:
+            d[i] = 1
+        else:
+            d[i] = d[i] + 1
+            
+    for key, value in d.items():
+        if max(d.values()) == value:
+            print(f'#{num} {key}')
```

Comment:
딕셔너리 활용을 정말 잘해주셨네요! 👍 
```suggestion
    result = []
    for key, value in d.items():
        if max(d.values()) == value:
            result.append(key)
    print(f'#{num} {max(result)}')
```
문제 조건에 **최빈수가 여러 개라면 가장 큰 수를 출력하라**는 조건이 있어요! 그 부분만 고려해주시면 완벽할 것 같습니다 😄